integrations/helpscout.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Helpscout importer'
+---
+
+To connect your HelpScout account, head to **Settings** → **HelpScout importer** and follow the instructions. 
+You can also sync multiple HelpScout mailboxes to Plain if you need to.
+
+
+When you connect your HelpScout account with Plain, we will import:
+
+- all your end users as Plain [**customers**](/api-reference/graphql/customers)
+- all your conversations as Plain [**threads**](/api-reference/graphql/threads). We will import all conversation threads as messages, and all internal notes. We'll map every HelpScout conversation status to an equivalent Plain thread status.
+- all the conversation tags as Plain [**labels**](/api-reference/graphql/labels)
+- all attachments that were previously added to conversations in HelpScout, will be added to threads in Plain 
+
+After you connect HelpScout for the first time, we will import all your existing data. 
+From that point, any new new end user, conversation, thread, note, or tag created in your 
+HelpScout account, will be automatically synced into Plain and kept up-to-date.
+
+To stop syncing data from HelpScout, click on **Settings** → **HelpScout importer** -> **Disconnect**.
+
+<Info>
+  The data created through the HelpScout importer will not trigger any webhooks or auto-responders.
+</Info>
+
+The HelpScout importer is available on [Scale tier](https://www.plain.com/pricing) only – please [reach out for a demo](https://38j36lhg2hq.typeform.com/to/zRvsFJPO) if your interested.
```

Comment:
Worth checking if this is how we want people to reach out. I think typeform might be legacy?

Reply:
It's still on the homepage